crypto/keyring_message.go

```diff
@@ -228,7 +233,14 @@ func asymmetricDecryptStream(
 		privKeyEntries = append(privKeyEntries, additionalEntries...)
 	}
 
-	config := &packet.Config{Time: getTimeGenerator()}
+	config := &packet.Config{
+		Time: func() time.Time {
+			if verifyTime == 0 {
+				return getNow()
```

Comment:
This is a bit nitpicky but it may also be worth adding a comment here saying that we ignore signature expiration errors later if `verifyTime == 0`, otherwise readers may get the impression that the intention is that `0` means "current time" (rather than "ignore time")